rd_ui/app/scripts/services/notifications.js

```diff
@@ -27,23 +30,13 @@
                 return;
             }
 
-            if (document.webkitVisibilityState && document.webkitVisibilityState == 'visible') {
```

Comment:
To emphasize the effect of notifying!
the truth is I removed it while testing to always see the notifications, and decided it's not that bad to keep it that way.
wdyt?